packages/@uppy/remote-sources/src/index.js

```diff
@@ -38,7 +37,6 @@ export default class RemoteSources extends BasePlugin {
 
     const defaultOptions = {
       sources: Object.keys(availablePlugins),
-      target: Dashboard,
```

Comment:
this is a breaking change, right?

Reply:
Hope we can figure out something without adding more options. Either by letting plugins do breaking changes whenever they want or by skipping this feature in remote sources for now.